packages/trader/src/Stores/Modules/Profit/profit-store.js

```diff
@@ -66,7 +66,9 @@ export default class ProfitTableStore extends BaseStore {
         const response = await WS.profitTable(
             batch_size,
             !should_load_partially ? this.data.length : undefined,
-            getDateBoundaries(this.date_from, this.date_to, this.partial_fetch_time, should_load_partially)
+            /* Last fetch time (`partial_fetch_time`) should not be set to `date_from` while fetching latest profit table records because 
+            date filtering happens based on `buy_time` of a contract. If we visit profit table and then sell a contract which was purchased earlier, that particular contract won't be visible in profit table when visited again. */
```

Comment:
can we break this long line into smaller :D

Reply:
Sure 👍